web_app/dist/main.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
Not something to address in this PR, but the three of us need to settle on an HTML whitespace (tabbing/ newlines) convention

Reply:
kk